### PR TITLE
Fetch data from api

### PR DESCRIPTION
**Pull request by ezraermy**
This is the 1rst project (fetch data from api) of the third module. I created a new component and added a logic using swr library to fetch data from an api.

## Essential requirements:
- Use Html/CSS
- Use JavaScript ES6
- reactJS
- api
- Make sure linter for free of errors.
- Using GitHub flow.
- Documented in a professional way.

### CODE REVIEW

File: src/components/quotes.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  if (!advice) return <div>Loading...</div>;
+
+  return (
+    <div className="quotes-box">
+      <p className="text">{advice.slip.advice}</p>
+    </div>
+  );
+}
+
+export default Quotes;
```
Comment (rotimiazeez) [inline src/components/quotes.js]: - Great job using SWR 👍🏽, but the project requires the use of `UseEffect.` for practice purposes. 😉
Screenshot from 2023-03-30 08-39-04

Comment (rotimiazeez) [reply src/components/quotes.js]: - Please ensure in your `app.js` The `<Quotes>` component should be imported next to `Mycalc.` 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @ezraermy 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: src/components/quotes.js (hunk 1 above)
Comment (ezraermy) [reply src/components/quotes.js]: @rotimiazeez Thank you for the well-detailed feedback. I will correct the code as you suggested.

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @ezraermy ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of GitFlow :+1: 
- Good job fetching the API :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**